packages/templating-tools/package.js

```diff
@@ -16,7 +16,7 @@ Package.onUse(function(api) {
     // boilerplate-generator uses spacebars-compiler.)
     // XXX maybe uglify should be applied by this plugin instead of via magic
     // weak dependency.
-    'minifier-js@1.2.14'
+    'minifier-js@1.2.14 || 2.0.0'
```

Comment:
I just finished running a search through the entire Blaze code, the minify function from the original `minifier-js` package isn't used anywhere. The reason the tests were failing originally is because the beautification part wasn't there, which was an issue with `spacebars-compiler`. The reason that `templating-tools` was failing is because while `spacebars-compiler` had failsafes in place in case it wasn't there, `templating-tools` did not. In my opinion that was the only reason the package was included as a strong dependency in `templating-tools` to begin with. It had nothing to do with minification, it was beautification.